assets/wire/components/ItemDetails.jsx

```diff
@@ -48,7 +48,7 @@ function ItemDetails({
     listConfig,
     filterGroupLabels,
 }) {
-    const picture = getPicture(item);
+    const picture = getPictureList(item);
```

Comment:
this should be rather called pictures now, and not sure if that `isCustomRendition` would work with a list?